zerver/lib/actions.py

```diff
@@ -2537,11 +2537,14 @@ def do_update_message(user_profile, message, subject, propagate_mode, content, r
         edit_history_event["prev_content"] = message.content
         edit_history_event["prev_rendered_content"] = message.rendered_content
         edit_history_event["prev_rendered_content_version"] = message.rendered_content_version
+        edit_history_event["edited_by"] = user_profile.email
         message.content = content
         message.rendered_content = rendered_content
         message.rendered_content_version = bugdown_version
         event["content"] = content
         event["rendered_content"] = rendered_content
+        event["edited_by"] = user_profile.email
+        event['prev_rendered_content_version'] = message.rendered_content_version
```

Comment:
We've been moving to a world where we keep track of users by user ID, not by email; can you change these to use user IDs?